hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/audit/AuditMessage.java

```diff
@@ -120,7 +120,7 @@ public Builder withException(Throwable ex){
     public AuditMessage build(){
       AuditMessage auditMessage = new AuditMessage();
       auditMessage.message = "user=" + this.user + " | ip=" + this.ip + " | " +
-          "op=" + this.op + " | " + "ret=" + this.ret;
+          "op=" + this.op + " " + this.params + " | " + "ret=" + this.ret;
```

Comment:
Missing params =

Reply:
@bharatviswa504 `params=` is not required. the old format was something like:
`..... | op=OPERATION_NAME [params] |......`
So changes from @adoroszlai are fine.

This seems to have been broken by HDDS-2323.

Reply:
My question was can we add like params = <<params>>. Like how for others we have.